### ISSUE TITLE
Stop Autoplaying Next VoD

### ISSUE THREAD

**Issue by Gelletti** [night/BetterTTV#2095]
Hi there,

tl;dr: One VoD is not the other, the first one makes you slumber, the next sounds like chainsawing lumber.

First of all, amazing work on your plugin, incredibly useful!
I use VoD when I go to bed and try to fall asleep.
My problem arises when I start watching a video of a streamer playing a casual slowpaced game solo.
The sounds are quiet, his voice is low, he's not getting excited and shouting.
Then I fall asleep.
When the video ends, the next (or a random?) video of his starts playing, where he might be together with his whole group of friends playing some FPS, with explosions, yelling, laughing and overall much higher volume.
I naturally wake up to this, but tend to be too groggy or half-asleep to actually get up and turn my computer off, or if I do, I'm quite awake by then, which defeats the whole purpose.

I've seen a couple of threads on this issue, but no solutions by anyone yet.
I was wondering if you could implement an option to stop the autoplay of VoDs after one ends.

Thanks for listening, whether you decide to implement anyhing like it or not!

Kind Regards,

Gelletti

**Comment by xgerhard** [comment]
Duplicate of #1996
